kafka-impl/src/main/java/io/streamnative/pulsar/handlers/kop/security/SaslAuthenticator.java

```diff
@@ -420,15 +420,9 @@ private static ByteBuf buildKafkaResponse(RequestHeader header,
 
     @VisibleForTesting
     public static ByteBuf sizePrefixed(ByteBuffer buffer) {
```

Comment:
I think we can remove this method and replace the usage with `Unpooled.wrappedBuffer(response)`.

Reply:
+1

Reply:
Yes, this method can be removed.

Reply:
I have addressed your comments. PTAL